bin/logify.pl

```diff
@@ -54,6 +54,10 @@ sub logLineForDeclaration {
 	my $innards = "%log; ";
 	if($rtype ne "void") {
 		$innards .= "$rtype r = %orig; ";
+		if ($rtype eq "instancetype") {
+			$innards .= "id r = %orig; ";
+		}
```

Comment:
This logic would cause a compile error from redefining `r`. How about:

```suggestion
		if ($rtype eq "instancetype") {
			$rtype = "id";
		}
		$innards .= "$rtype r = %orig; ";
```

Reply:
It's almost the same. The latter request is together